lib/ansible_tower_client/base_models/project.rb

```diff
@@ -17,5 +17,10 @@ def update
 
       api.project_updates.find(update['project_update'])
     end
+
+    def last_update
+      return if related['last_update'].blank?
+      api.project_updates.find(related['last_update'])
+    end
```

Comment:
I'm confused.  Is this method called?  I don't see it.

Reply:
The other changes look good. 👍

Reply:
It is not called in the API itself. The `manageiq-providers-ansible_tower` will use it.

Reply:
I see calls to `last_updated` but not `last_update` in the https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/72 that uses this code change.  Is this right or a typo?  It looks like it's being called on a `project`.

Reply:
I'm fine with adding new methods but have seen many times where we add code we are sure we'll need but never end up using.  I want to confirm it's actually going to be used next.

Finally, if you're comfortable with `git rebase -i`, it would be nice to squash some of the commits with "fix" or "style" in the name so each commit is standalone with a single type of change.  I don't personally like squashing all commits, but minor typos/style/fix commits are definitely worth squashing.

Reply:
The `project.last_updated` in [ManageIQ/manageiq-providers-ansible_tower#72](https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/72) is really a typo. Thanks for finding out! I’ll fix it along with the other changes requested for that PR.

I’ll do the squashing as you suggested.

Reply:
Don't worry about squashing commits if it becomes difficult.  It's generally good practice to commit "oops/typos/fixes" separate from logical change commits and forcing yourself to try to do this makes it easier to squash commits.  Ping me when it's ready to go.